src/components/UserPortal/OrganizationCard/OrganizationCard.tsx

```
@@ -100,96 +131,113 @@ function organizationCard(props: InterfaceOrganizationCardProps): JSX.Element {
+              dataTestId="emptyContainerForImage"
+            />
+          )}
+        </div>
+        <div className={styles.content}>
+          <Tooltip title={name} placement="top-end">
+            <h4 className={`${styles.orgName} fw-semibold`}>{name}</h4>
+          </Tooltip>
+          <h6 className={`${styles.orgdesc} fw-semibold`}>
+            <span>{description}</span>
+          </h6>
+          {address && address.city && (
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)